docs/cookbook/html-conversion.md

```diff
@@ -207,8 +207,8 @@ Next, we need to add the schema fields to populate our navigation menu. If your
 module.exports = {
   fields: {
     add: {
-      homePage: {
-        label: 'Page to link',
+      _homePage: {
```

Comment:
Apostrophe automatically makes the home page available as `data.home`, so I don't think it makes sense to ask users to manually pick it in the global doc.

https://v3.docs.apostrophecms.org/cookbook/building-navigation.html#generating-site-navigation-from-the-page-tree